taxonomies/categories.txt

```diff
@@ -68394,7 +68399,7 @@ wikidata:Q1886522
 en:Pickles
 ca:Conserves
 da:Syltede agurker
```

Comment:
```suggestion
```
This also means "pickled cucumbers", so could move to the en:Pickled cucumbers section around line 75405.

Reply:
@svensven : thanks, I just moved it.